Wikipedia/Code/NotificationsCenterOnboardingView.swift

```diff
@@ -0,0 +1,170 @@
+                        .foregroundColor(Color(theme.colors.primaryText))
+                    Spacer(minLength: 50)
+                    NotificationsCenterOnboardingRowView(theme: theme, image: .notificationsAndAlerts, header: LocalizedStrings.notificationsAlertsTitle, message: LocalizedStrings.notificationsAlertsMessage)
+                    Spacer(minLength: 8)
+                    NotificationsCenterOnboardingRowView(theme: theme, image: .filter, header: LocalizedStrings.filterTitle, message: LocalizedStrings.filterMessage)
+                    Spacer(minLength: 8)
+                    NotificationsCenterOnboardingRowView(theme: theme, image: .inbox, header: LocalizedStrings.inboxTitle, message: LocalizedStrings.inboxMessage)
+                    Spacer(minLength: 8)
+                    NotificationsCenterOnboardingRowView(theme: theme, image: .push, header: LocalizedStrings.pushTitle, message: LocalizedStrings.pushMessage)
+                }
+                .padding()
+            }
+            ZStack(alignment: .bottom, content: {
+                VStack {
+                    Button(action: {
+                        dismissAction?()
+                    }, label: {
+                        Text(LocalizedStrings.continueButton)
+                            .font(.body.bold())
+                            .foregroundColor(continueButtonTextColor)
+                            .padding()
+                            .frame(maxWidth: .infinity)
+                            .background(Color(theme.colors.link))
+                            .cornerRadius(8)
+                            .padding()
+                    })
+                    Spacer().frame(height: 8)
+                    Button(LocalizedStrings.learnMoreButton) {
+                        userDidTapLearnMore()
+                    }
+                        .font(.body.weight(.semibold))
+                        .foregroundColor(Color(theme.colors.link))
+                    Spacer().frame(height: 18)
+                }
+            })
+        }
+    }
+
+    // MARK: - Actions
+
+    func userDidTapLearnMore() {
+        // TODO: Replace with actual Meta Wiki FAQ when available
+        guard let url = URL(string: "https://phabricator.wikimedia.org/T302634") else {
+            return
+        }
+
+        UIApplication.shared.open(url)
+    }
+
+}
+
+fileprivate struct NotificationsCenterOnboardingRowView: View {
+
+    // MARK: - Properties
+
+    var theme: Theme
+    var image: OnboardingRowIconImage
+    var header: String
+    var message: String
+
+    // MARK: - Content
+
+    var body: some View {
+        HStack(alignment: .top) {
+            image.imageForTheme(theme: theme)
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(maxWidth: 28, maxHeight: .infinity, alignment: .top)
+                .offset(y: 10)
+                .accessibility(label: image.accessibilityLabel)
```

Comment:
Yeah this one tripped me up too! I think the reason is that `Text` does localizable string lookups by default - its [default initialization](https://developer.apple.com/documentation/swiftui/text/init(_:tablename:bundle:comment:)) hides that away a bit. So even though in this accessibility context it doesn't necessarily visually render `Text`, it can still benefit from the under the hood default localization lookups `Text` uses.

Reply:
Ah, that makes sense - thanks!